lib/supabase/in_process_tokens/getMaxBlockTimestamp.js

```diff
@@ -26,7 +26,7 @@ export async function getMaxBlockTimestamp() {
     };
   });
 
-  const maxTimestamps = await Promise.all(promise);
+  const maxTimestamps = await Promise.all(promises);
```

Comment:
same thing here for single responsibility principle
- actual: tightly coupled supabase select call with logic for maxTimestamp logic
- required: new lib for lib/moments/getMaxBlockTimestamp.js which calls a supabase lib to selectMoments.